### ISSUE TITLE
Bring back "more" button in add bookmark drop-down

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#34878]
## Description 

Several users have noticed that the `More` button no longer appears when trying to add a new bookmark in the browser. This requires users to perform additional clicks in order to change the folder in which they want to save the bookmark to. I suggest re-adding the `More` button (or equivalent) to the bookmark drop-down for ease of access or the user. 

https://community.brave.com/t/adding-a-bookmark-doesnt-give-complete-dropdown-menu-choices-like-before/521260
https://community.brave.com/t/save-bookmarks-option/520165/6

**Comment by aguscruiz** [comment]
@rebron is this one still needed? I see the folder options now in the bookmarks dropdown

**Comment by Brave-Matt** [comment]
Yes — this is still desired. I personally am fine using the drop down but there seem to be a good chunk of users who really want the this particular button back:
https://community.brave.com/t/save-bookmarks-option/520165/11
